hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/conf/TestCommonConfigurationFields.java

```diff
@@ -142,6 +141,7 @@ public void initializeMemberVariables() {
     xmlPropsToSkipCompare.add("fs.viewfs.overload.scheme.target.webhdfs.impl");
     xmlPropsToSkipCompare.add("fs.viewfs.overload.scheme.target.wasb.impl");
     xmlPropsToSkipCompare.add("fs.viewfs.overload.scheme.target.swift.impl");
+    xmlPropsToSkipCompare.add("fs.viewfs.overload.scheme.target.gs.impl");
```

Comment:
Seems like this list should be sorted alphabetically?

Reply:
@medb I can certainly change this, but it's already not sorted due to `swift`.

Reply:
yeah, but lets try not to make things worse. It's like how we try and follow ordering/layout rules on imports. Strive to to better than your predecessors

Reply:
@steveloughran done, also fixed the `swift` order.